hackathon/templates/hackathon/create-event.html

```diff
@@ -37,10 +37,13 @@ <h1>Create Hackathon</h1>
         <div class="col-12">
           {{ form.organisation|as_crispy_field }}
         </div>
-        <div class="col-12 col-md-6">
+        <div class="col-12 col-md-6 mb-3">
+          {{ form.teamsize|as_crispy_field }}
```

Comment:
`{{ form.team_size|as_crispy_field }}`
(ie: `team_size`) ?

Reply:
Updated to `team_size`